scripts/graphql_api_content/render_helpers.rb

```diff
@@ -213,6 +213,8 @@ def render_page(schema_type_data)
 
     page_content = <<~HTML.strip
       ---
+      title: #{name} | #{namespace} | GraphQL API Reference
```

Comment:
Any particular reason for using `|` between each? I was expecting `-` and the just `|` between this and `Buildkite Documentation` (For example: `agent - Queries - GraphQL API | Buildkite Documentation`). Maybe that's unnecessarily complicated.

I also think we can drop "Reference" to keep the title slightly shorter. Thoughts?

Reply:
No reason other than for aesthetics! I'll update.

Reply:
OK cool, I've updated the formatting in https://github.com/buildkite/docs/pull/2376/commits/193b445aaa15ee6b1a0a430e5efa706059568837. Worth noting ` | Buildkite Documentation` is appended to all page titles which is why it's not included in that `title` attribute. 

https://github.com/buildkite/docs/blob/193b445aaa15ee6b1a0a430e5efa706059568837/app/views/application/_head.html.erb#L8